### PR TITLE
[CELEBORN-1317][FOLLOWUP] Improve parameters, description and document of REST API

### PR DESCRIPTION
**Pull request by SteNicholas**
### What changes were proposed in this pull request?

Improve parameters, description and document of Celeborn REST API, including:

1. The POST request uses `@FormParam` instead of `@QueryParam`.
2. The parameter name uses lowercase instead of uppercase.
3. The description of `/exclude` aligns with document in `monitoring.md`.
4. The document of `REST API` adds the `Method` and `Parameters` to document GET/POST method and corresponding interface.

### Why are the changes needed?

The parameters, description and document of REST API need to improve after http server refine.

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

GA.

### CODE REVIEW

**Comment by SteNicholas** [comment]
Ping @RexXiong, @turboFei.

**Comment by SteNicholas** [comment]
Ping @RexXiong, @turboFei.

**Comment by SteNicholas** [comment]
Ping @turboFei, @RexXiong. PTAL.

File: master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/ApiMasterResource.scala [hunk 1]
```diff
@@ -102,11 +88,26 @@ class ApiMasterResource extends ApiRequestContext {
...
-    description = "List all worker event infos of the master.")
+    description =
+      "Excluded workers of the master add or remove the worker manually given worker id. The parameter add or remove specifies the excluded workers to add or remove, which value is separated by commas.")
   @POST
-  def excludeWorkers(
-      @QueryParam("ADD") addWorkers: String,
-      @QueryParam("REMOVE") removeWorkers: String): String = {
+  def exclude(
+      @FormParam("add") addWorkers: String,
+      @FormParam("remove") removeWorkers: String): String = {
```
[the remark this answers is not in the document]
Comment (SteNicholas) [reply master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/ApiMasterResource.scala:96]: @turboFei, `FormParam` is case-sensitive. parameters are case-sensitive in the JAX-RS specification. Therefore, do we need to support case insensitive parameters?

[the remark this answers is not in the document]
Comment (turboFei) [reply master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/ApiMasterResource.scala:96]: ok, I just think that it is a break change for this API.

Has this api been released before? If that, I think we need to mention it in migration user guide.

[the remark this answers is not in the document]
Comment (SteNicholas) [reply master/src/main/scala/org/apache/celeborn/service/deploy/master/http/api/ApiMasterResource.scala:96]: @turboFei, I have mentioned the break change of REST API `/exclude` and `/exit` in `migration.md` including POST method and form parameter. PTAL.

**Review by turboFei — APPROVED** [review]

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

**Pull request closed without merging**